main/lsp/ErrorReporter.h

```diff
@@ -0,0 +1,40 @@
+#ifndef RUBY_TYPER_LSP_ERRORREPORTER_H
+#define RUBY_TYPER_LSP_ERRORREPORTER_H
+
+#include "ast/ast.h"
```

Comment:
I think we might not need this import; this file doesn't reference anything in `ast` AFAIK.

Reply:
that was left over from some previous changes, my bad